source/client/output_formatter_impl.h

```diff
@@ -108,5 +108,10 @@ class FortioOutputFormatterImpl : public OutputFormatterImpl {
   double durationToSeconds(const Envoy::ProtobufWkt::Duration& duration) const;
 };
 
+class FortioPedanticOutputFormatterImpl : public FortioOutputFormatterImpl {
```

Comment:
Can we add a comment on the class roughly explaining what it is for?

Reply:
Done in 2df8c2be